index.js

```diff
@@ -1,2 +1 @@
  
```

Comment:
Thank you for adding a new line here, it helps to identify the initialization phase of the variables

Reply:
Ack!